ruby_exercises/mythical-creatures/lib/centaur.rb

```diff
@@ -0,0 +1,39 @@
+class Centaur
+  attr_reader :name,
+              :breed,
+              :shoot,
+              :run,
+              :cranky,
+              :standing
+      
+
+  def initialize(name, breed = 'Palomino')
+    @name = name
+    @breed = breed
+    @shoot = 'Twang!!!'
```

Comment:
I would say that `run` and `shoot` shouldn't be instance variables -- they're not attributes of a Centaur but actions that a Centaur does. I would move their values to the methods below.